opentelemetry-sdk/src/metrics/internal/sum.rs

```diff
@@ -29,12 +43,41 @@ impl<T: Number<T>> Default for ValueMap<T> {
 
 impl<T: Number<T>> ValueMap<T> {
     fn new() -> Self {
+        let buckets = std::iter::repeat_with(|| Mutex::new(None))
+            .take(BUCKET_COUNT)
+            .collect::<Vec<_>>()
+            .try_into()
+            .unwrap_or_else(|_| panic!("Incorrect length"));
```

Comment:
you'll remove the panic calls right?

Reply:
this will never reach, as the bucket count is fixed. Just kept for readability.

Reply:
ok this is removed now.